source/dem/dem.cc

```diff
@@ -175,8 +174,6 @@ DEMSolver<dim>::DEMSolver(DEMSolverParameters<dim> dem_parameters)
       throw std::runtime_error("Specified load balance method is not valid");
     }
 
-
-
   // Calling input_parameter_inspection to evaluate input parameters in the
   // parameter handler file, finding maximum particle diameter used in
   // polydisperse systems
```

Comment:
Can you update the dem.cc file to use load_balance_particle_weight parameter, instead of the hard-coded particle_weight = 10000?

Reply:
done